drivers/mysql.go

```diff
@@ -373,19 +373,19 @@ func (db *MySQL) GetRecords(database, table, where, sort string, offset, limit i
 		return nil, 0, err
 	}
 
-	return
+	return paginatedResults, totalRecords, nil
 }
 
-func (db *MySQL) ExecuteQuery(query string) (results [][]string, err error) {
+func (db *MySQL) ExecuteQuery(query string) (results [][]string, records int, err error) {
```

Comment:
This name is used in previous method, I find it more meaningful

```suggestion
func (db *MySQL) ExecuteQuery(query string) (results [][]string, totalRecords int, err error) {
```

Reply:
I disagree. The other method (`GetRecords`) expects an `offset` and a `limit` and so `totalRecords` is the result of doing the same query but then without the `offset` and `limit` (`SELECT COUNT(*) FROM TABLE`) so that the caller can see/know how many records there are in total which may not the amount of records that are returned.

The `ExecuteQuery` method on the other hand just executes a given query and returns the result. So in this case the returned number of records is just the number of records that you are going to receive. Even if you've send a query which included an `offset` and a `limit`. So calling it `totalRecords` here as well is a bit misleading in my opinion.

Reply:
You are right.

My misunderstanding is indeed linked to my other comment about the -1